.gitignore

```diff
@@ -27,3 +27,9 @@
 *.exe
 *.out
 *.app
+
+#Temporary files
+~*
+
+#Text files
+*.txt
```

Comment:
Text files are really important for us. Look at the asm/tables.txt. We shouldn't ignore it.